frontend/src/i18n/en.json

```diff
@@ -214,6 +215,7 @@
     "rules": "Rules",
     "rulesHelp": "Here you can define a set of allow and disallow rules for this specific user. The blocked files won't show up in the listings and they wont be accessible to the user. We support regex and paths relative to the users scope.\n",
     "scope": "Scope",
+    "setDateFormat": "Set exact date format",
```

Comment:
duplicate of line 30

Reply:
Actually, I copy the feture "hit dot files" to implement the issue.
there are two "hit dot files" settings in en.json
I'll commit that